src/mach/mod.rs

```diff
@@ -502,6 +529,11 @@ pub enum Mach<'a> {
 impl<'a> Mach<'a> {
     /// Parse from `bytes` either a multi-arch binary or a regular mach-o binary
     pub fn parse(bytes: &'a [u8]) -> error::Result<Self> {
+        Self::parse_2(bytes, false)
+    }
+
+    /// Parse from `bytes` either a multi-arch binary or a regular mach-o binary
+    pub fn parse_2(bytes: &'a [u8], lossy: bool) -> error::Result<Self> {
```

Comment:
ditto; i'd make this function, `fn parse_impl` or something, then have:
```rust
pub fn parse()

pub fn parse_lossy()
```

call the `impl` with the boolean value forwarded